pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -152,6 +152,45 @@ func (k *Kubernetes) InitDS(name string, service kobject.ServiceConfig) *extensi
 	return ds
 }
 
+func (k *Kubernetes) initIngress(name string, service kobject.ServiceConfig, port int32) *extensions.Ingress {
+
+	ingress := &extensions.Ingress{
+		TypeMeta: unversioned.TypeMeta{
+			Kind:       "Ingress",
+			APIVersion: "extensions/v1beta1",
+		},
+		ObjectMeta: api.ObjectMeta{
+			Name: name,
+		},
+		Spec: extensions.IngressSpec{
+			Rules: []extensions.IngressRule{
+				{
+					IngressRuleValue: extensions.IngressRuleValue{
+						HTTP: &extensions.HTTPIngressRuleValue{
+							Paths: []extensions.HTTPIngressPath{
+								{
+									Backend: extensions.IngressBackend{
+										ServiceName: name,
+										ServicePort: intstr.IntOrString{
+											IntVal: port,
+										},
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+		},
+	}
+
+	if service.ExposeService != "true" {
```

Comment:
In that case, `false` gets treated as a hostname. If the user does not want to expose, he would rather not write (or comment) this in the compose file.

Even I am divided on this. Should we enforce, (true, false, hostname), or just (true, hostname)?